commit d26f3118273af4b5817a71fbf4c420a0e7505616
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Mar 8 11:09:49 2019 +0000

    staging: vc-sm-cma: Don't fail if debugfs calls fail.
    
    Return codes from debugfs calls should never alter the
    flow of the main code.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
index c77d2088b2f5..8ae7810f1f9a 100644
--- a/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
+++ b/drivers/staging/vc04_services/vc-sm-cma/vc_sm.c
@@ -681,13 +681,6 @@ static void vc_sm_connected_init(void)
 
 	/* Create a debug fs directory entry (root). */
 	sm_state->dir_root = debugfs_create_dir(VC_SM_DIR_ROOT_NAME, NULL);
-	if (!sm_state->dir_root) {
-		pr_err("[%s]: failed to create \'%s\' directory entry\n",
-		       __func__, VC_SM_DIR_ROOT_NAME);
-
-		ret = -EPERM;
-		goto err_stop_sm_service;
-	}
 
 	sm_state->dir_state.show = &vc_sm_cma_global_state_show;
 	sm_state->dir_state.dir_entry =
@@ -720,7 +713,6 @@ static void vc_sm_connected_init(void)
 
 err_remove_shared_memory:
 	debugfs_remove_recursive(sm_state->dir_root);
-err_stop_sm_service:
 	vc_sm_cma_vchi_stop(&sm_state->sm_handle);
 err_failed:
 	pr_info("[%s]: failed, ret %d\n", __func__, ret);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1529_d26f3118273af4b5817a71fbf4c420a0e7505616.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1529_d26f3118273af4b5817a71fbf4c420a0e7505616.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
